libs/ai-endpoints/pyproject.toml

```diff
@@ -26,7 +26,7 @@ pytest-mock = "^3.10.0"
 syrupy = "^4.0.2"
 pytest-watcher = "^0.3.4"
 pytest-asyncio = "^0.21.1"
-langchain-core = { git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/core", tag = "langchain-core==0.2.40" }
```

Comment:
wondering if we still want to install from source for test dependencies (e.g., remove specification of `tag`)

Reply:
i agree, i'll make the change.